class DataNotFoundError(Exception):
    pass

class DataExistError(Exception):
    pass
